app/code/community/Bolt/Boltpay/sql/bolt_boltpay_setup/upgrade-0.0.17-1.1.0.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Magento
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@magento.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade the Bolt extension
+ * to a newer versions in the future. If you wish to customize this extension
+ * for your needs please refer to http://www.magento.com for more information.
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (http://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * This installer adds the Bolt order token and parent quote id for bolt order tracking.
+ */
+/* @var Mage_Sales_Model_Resource_Setup $installer */
+$installer = Mage::getResourceModel('sales/setup', 'sales_setup');
+$installer->startSetup();
+
+Mage::log('Installing Bolt 1.1.0 updates', null, 'bolt_install.log');
+
+$installer->addAttribute(
+    "quote", "bolt_order_token", array(
```

Comment:
unused?

Reply:
Ultimately, if the token returned from the Bolt API create order endpoint is different from the token sent in a Bolt API fetch call, there will be no real need for this.  If they are the same, then it will be used as a nonce and is a better way to look up quotes as the sequential quote id/order reference is many times easier to guess.